common/src/main/java/com/wynntils/features/inventory/CustomBankPagesFeature.java

```diff
@@ -117,51 +69,31 @@ private void getCustomJumpDestinations() {
             }
         }
 
-        customJumpDestinations = parseStringToDestinations(configDestinations, currentContainer);
+        customJumpDestinations = parseStringToDestinations(configDestinations, containerType);
 
         if (customJumpDestinations == null) {
-            customJumpDestinations = getDefaultJumpDestinations();
+            customJumpDestinations = Models.ContainerQuickJump.getJumpDestinations(containerType);
         }
     }
 
-    private List<Integer> getDefaultJumpDestinations() {
-        return switch (currentContainer) {
-            case BANK -> QUICK_JUMP_DESTINATIONS;
-            case BLOCK_BANK -> BLOCK_BANK_DESTINATIONS;
-            default -> HOUSING_DEFAULT_DESTINATIONS; // this has the lowest values, so it's the safest default
-        };
-    }
-
-    @SubscribeEvent
-    public void onScreenClose(ScreenClosedEvent e) {
```

Comment:
This is required, otherwise when opening a different container from the bank screen the jump buttons are still visible, although due to your changes it instead throws an error. Right click potions whilst in a bank screen and you'll see it

Reply:
works perfectly fine but ill add a listener in the quick jump model

Reply:
I'm still getting `Index 82 out of bounds for length 63`

Reply:
Actually, seems like this is an issue outside of this pr.